---
layout: lesson
root: .  # Is the only page that don't follow the partner /:path/index.html
permalink: index.html  # Is the only page that don't follow the partner /:path/index.html
---

__This workshop is in a DRAFT stage__

**Synopsis:** Introduces the first steps of analysing RNA sequencing data using the command line - including how to pre-process raw sequencing reads to analysis ready count data. We will use the [nf-co.re's rnaseq workflow](https://nf-co.re/rnaseq/usage) on Pawsey Nimbus Cloud.

**Target audience:** Researchers, students and life scientists who are new to RNA sequencing for differential expression and pathway analysis. The course is beginner friendly and intended for those interested in using the command line interface for their analysis. It is also suitable for those who want to learn about and use [nf-co.re](https://nf-co.re/) workflows.

**Follow-on courses:** This course is __part one__ of the two-part __[RNA sequencing training series](https://sydney-informatics-hub.github.io/training.RNAseq.series/)__. The second part of the series uses R/R Studio on Pawsey Nimbus Cloud to perform differential expression and pathway analysis using raw count data generated in part 1.

We recommend attending both courses if you would like and end to end overview of how to perform the analysis. If you are only interested in the technologies used, you may be able to get away with attending only the parts of the workshop of interest :smiley:. 


> ## Prerequisites
> * Familiarity with [Unix/Linux command line](https://datacarpentry.org/shell-genomics/)
> * **Own laptop and access to Pawsey Nimbus is required**

{: .prereq}


{% include links.md %}

